www/src/routes/student/resumeReview/ResumeReviewCard.tsx

```diff
@@ -53,6 +55,8 @@ const ResumeReviewCard: FC<ResumeReviewCardProps> = ({ resumeReview }: ResumeRev
         ['seeking_reviewer', 'Pending'],
     ]);
 
+    const actionsShouldBeDisabled = !userContext?.hasAgreedToTermsOfService ?? false;
```

Comment:
Are `!userContext?.hasAgreedToTermsOfService ?? false` and `!(userContext?.hasAgreedToTermsOfService ?? false)` equivalent? Lol sorry for slightly dumb question just wanna double check the logic is the same as the MockInterview

Reply:
Ah no good catch this should be parenthesized before negated lol

Reply:
Fixed 👍